tests/backup/backup_test.go

```diff
@@ -2661,6 +2661,263 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 	})
 })
 
+func GetScheduleUID(backupName string, orgID string, ctx context.Context) (string, error) {
+	backupDriver := Inst().Backup
+	bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
+		Name:  backupName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+	resp, err := backupDriver.InspectBackupSchedule(ctx, backupSchInspectRequest)
+	log.FailOnError(err, "Failed for req - %v", backupSchInspectRequest)
```

Comment:
Can we change `logFailOnError()` to a `return` as this function has return and change message as well
```
return "", fmt.Errorf"Failed to inspect backup [%s] with UID [%s], Err: %v", backupName, bkpUid, err)
```
Lets not mix `logFailOnError()` and `return`, just return and do `logFailOnError()` where you call that function, if you need to